tests/console/validate_repos.pm

```diff
@@ -42,4 +42,9 @@ sub run {
     die("Unexpected repository found: $unexpected_aliases") if $unexpected_aliases;
 }
 
+sub post_fail_hook {
+    select_console 'root-console';
+    upload_logs("/var/log/distro_migration.log", failok => 1);
+}
+
```

Comment:
this makes no sense, due to it is running in scenarios that are not migrations.

Reply:
updated, now only with the change for 'move validate-product as first'.